.github/workflows/package.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           - os: windows
             runner: windows-2019
           - os: macos
-            runner: macos-10.11
+            runner: macos-10.12
```

Comment:
Macos 10 is deprecated https://github.com/actions/runner-images

Reply:
I changed to latest (currently 11)